user_docs/installing_software_in_OpenSARlab.md

```diff
@@ -0,0 +1,23 @@
+[![OpenSARlab Header](assets/OSL_user_guide_header.png)](OpenSARlab_user_guide.md)
+
+# Installing Software in OpenSARlab
+
+## pip
+You can install pip packages to your /home/jovyan/.local/lib/python3.7/site-packages directory with the following command:
+- python -m pip install --user package_name
+
+It is important to use the  --user argument. This will install the package to your .local directory, avoiding any permission issues, and insuring that the package installation will persist when the OpenSARlab server restarts (so you don't have to reinstall it every time). 
+
+Note that pip packages in the .local directory will supersede other versions of the same package installed elsewhere.   
+
+## conda 
```

Comment:
`--use-local` will install locally built packages - not locally installed remote packages.

Updating or installing the system conda packages works most of the time. The problem is that that changes are not persistent. To be persistent, the packages need to be installed into the home directory and used somehow.

To install within the home directory (not in the system directory), first create a local conda directory, install into that, then either activate that local directory (can cause issues in notebooks) or do the absolute path to the library wanted.

```
conda create -p ~/.my-local-conda-dir
conda install -p ~/.my-local-conda-dir my-favorite-package
~/.my-local-conda-dir/bin/python3.8 -version
```

All of this is rather messy. Should we still add this section?